### PR TITLE
add movement to background nodes

### PR DESCRIPTION
**Pull request by PeterJFB**
Add movement to gray nodes in background

Light theme

https://user-images.githubusercontent.com/42469466/103674506-21b6c880-4f7f-11eb-9361-189b0ca609f4.mp4

Dark theme

https://user-images.githubusercontent.com/42469466/103674538-28ddd680-4f7f-11eb-9621-1702febfed85.mp4

Reloading page, resizing and mobile

https://user-images.githubusercontent.com/42469466/103674673-5165d080-4f7f-11eb-878a-cdf1e14c85b7.mp4

### CODE REVIEW

File: app/components/Header/FancyNodesCanvas.js [hunk 1]
```diff
@@ -63,15 +65,20 @@ class FancyNodesCanvas extends Component<Props, State> {
...
-    drawFancyNodes(context, {
-      width: this.state.width,
-      height: this.props.height,
+    if (this.state.animation) global.clearInterval(this.state.animation);
+
+    this.setState({
+      animation: drawFancyNodes(context, {
+        width: this.state.width,
+        height: this.props.height,
+      }),
```
Comment (LudvigHz) [inline app/components/Header/FancyNodesCanvas.js:74]: It was a bit confusing to me at first why you needed the animation in state, could you call it something else, perhaps `animationInterval`?.

**Review by LudvigHz — APPROVED** [review]
This is a really cool feature!
LGTM :shipit:

File: app/components/Header/FancyNodesCanvas.js (hunk 1 above)
Comment (PeterJFB) [reply app/components/Header/FancyNodesCanvas.js]: Good idea, i'll add it^^

**Review by ingraso — APPROVED** [review]
Looks very nice! Good work :smiley: :clap:

**Comment by LudvigHz** [comment]
@PeterJFB Was there something you wanted to fix on this or can we merge it?

**Comment by PeterJFB** [comment]
> @PeterJFB Was there something you wanted to fix on this or can we merge it?

Turns out `animationInterval` doesn't change back to `undefined` when using `clearInterval()`. I added it so you can easily check if the animation is running in the future. 
https://github.com/webkom/lego-webapp/blob/51310e2f4ff572de0154b37f44678d2df09c9f96/app/components/Header/FancyNodesCanvas.js#L84
Everything is good to go.

**Review by AdrianAndersen — APPROVED** [review]
This is a really cool feature. :100: Good job! :shipit:

**Pull request merged**